tests/test_dmxrange_add_sub.py

```diff
@@ -0,0 +1,47 @@
+"""Test functions for adding and removing DMX ranges.""" 
+
+import pytest
+
+from pint.models.dispersion_model import DispersionDM, DispersionDMX
+
+def test_startMJD_greaterThan_endMJD():
+    """ Check for error when start MJD is larger than end MJD. """
+    with pytest.raises(ValueError):
+        dm_mod=DispersionDMX()
```

Comment:
This should be outside the `with` - if a ValueError arises here, it's a sign the code is broken.

Reply:
fixed.